build.sbt

```diff
@@ -32,12 +49,9 @@ lazy val bsp4s = project
   .settings(
     Test / publishArtifact := false,
     Compile / doc / sources := Nil,
-    addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full),
     libraryDependencies ++= List(
-      "io.circe" %% "circe-core" % "0.9.0",
-      "io.circe" %% "circe-derivation" % "0.9.0-M4",
-      "io.circe" %% "circe-generic-extras" % "0.9.0",
-      "org.scalameta" %% "lsp4s" % "0.2.0"
+      "me.vican.jorge" %% "jsonrpc4s" % "ae3705e0+20210611-1304",
```

Comment:
- [ ] Update after jsonrpc4s is released.

This is a local version that I was working with. I made a pull request here: https://github.com/jvican/jsonrpc4s/pull/3 that can hopefully be merged and released. The current released version is missing some commits and differs a bit from what's on the master branch there atm. I couldn't get tests to pass without using those commits.